### PR TITLE
Created a new tool for creating Incidents from external systems

### PR DESCRIPTION
**Pull request by samikroy**
A detailed explanation is here.
https://github.com/samikroy/Azure-Sentinel/blob/patch-43/Playbooks/Create%20Incidents%20From%20Http/Readme.md

This will be a part of the readme after the PR merge.

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @samikroy, can you please share some working images of playbook.

**Comment by samikroy** [comment]
> Hello @samikroy, can you please share some working images of playbook.

Thank you for the review @v-prasadboke .

Please let me know if this helps 

image

image



image

File: Playbooks/Create Incidents From Http/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Generate-Incident-Logic-App",
+        "description": "This playbook will help to create an incident in Microsoft Sentinle when an email is sent to the configured email address.",
+        "prerequisites": [
```
Comment (v-prasadboke) [inline Playbooks/Create Incidents From Http/azuredeploy.json:7]: Please add post deployment like prerequisites

Comment (v-prasadboke) [reply Playbooks/Create Incidents From Http/azuredeploy.json:7]: image

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**